WordPress/build.gradle

```diff
@@ -253,8 +247,9 @@ android {
 
         // Exclude React Native's JSC and Hermes debug binaries
         exclude '**/libjscexecutor.so'
-        exclude '**/libhermes-inspector.so'
-        exclude '**/libhermes-executor-debug.so'
+        // TO-DO How to exclude this for release builds
```

Comment:
Todo: How to exclude this in release builds? If we exclude them like this, hermes is no longer being included in the build.